langlib/langlib-test/src/test/resources/test-src/valuelib_test.bal

```diff
@@ -520,13 +530,13 @@ type StringArray string[];
 function testCloneWithTypeStringArray() {
    string anArray = "[\"hello\", \"world\"]";
    json j = <json> anArray.fromJsonString();
-    string[]|error cloned = j.cloneWithType(StringArray);
-    assert(cloned is string[], true);
+// string[]|error cloned = j.cloneWithType(StringArray);
+// assert(cloned is string[], true);
 
-    string[]  clonedArr= <string[]> a2;
-    assert(clonedArr.length(), anArray.length());
-    assert(clonedArr[0], "Hello");
-    assert(clonedArr[1], "World");
+// string[]  clonedArr= <string[]> a2;
+// assert(clonedArr.length(), anArray.length());
+// assert(clonedArr[0], "Hello");
+// assert(clonedArr[1], "World");
```

Comment:
any reason for comment-out these?

Reply:
a2 is an unrecognized symbol, I contacted the person who introduced the change and they said they ll fix it.

Reply:
ack. 
Please create an issue, and assigned it to the person working on it..